acceptance/openstack/containerinfra/v1/clustertemplates_test.go

```diff
@@ -40,32 +41,34 @@ func TestClusterTemplatesCRUD(t *testing.T) {
 	th.AssertNoErr(t, err)
 	th.AssertEquals(t, clusterTemplate.UUID, template.UUID)
 
-	/*
-		// Test cluster update
-		updateOpts := []clustertemplates.UpdateOptsBuilder{
-			clustertemplates.UpdateOpts{
-				Path:  "/master_lb_enabled",
-				Value: "false",
-				Op:    "replace",
-			},
-			clustertemplates.UpdateOpts{
-				Path:  "/registry_enabled",
-				Value: "false",
-				Op:    "replace",
-			},
-			clustertemplates.UpdateOpts{
-				Path:  "/labels/test",
-				Value: "test",
-				Op:    "add",
-			},
-		}
+	// Test cluster update
+	updateOpts := []clustertemplates.UpdateOptsBuilder{
+		clustertemplates.UpdateOpts{
+			Op:    clustertemplates.ReplaceOp,
+			Path:  "/master_lb_enabled",
+			Value: "false",
+		},
+		clustertemplates.UpdateOpts{
+			Op:    clustertemplates.ReplaceOp,
+			Path:  "/registry_enabled",
+			Value: "false",
+		},
+		clustertemplates.UpdateOpts{
+			Op:    clustertemplates.AddOp,
+			Path:  "/labels/test",
+			Value: "test",
+		},
+		clustertemplates.UpdateOpts{
+			Op:   clustertemplates.AddOp,
+			Path: "/labels/test1",
```

Comment:
The addition of this is now causing the test to fail.

If you wanted to test for this error, it might be best to do it in a separate `Update` call so you can test for success in one call and test for an error in another call.

Reply:
This got in by err. Fixed by removing this block.